samples/computing-providers/get_instance_resources.py

```diff
@@ -0,0 +1,23 @@
+import pprint
+from swan import Orchestrator
+
+def orchestrator_cp_resources(avaliability:bool=True) -> list[dict]:
+    """Retrieve currently avaliable orchestrator cp resources (instance).
+
+    Args:
+        avalibility: True for availiable instance configuration only; False for all configuration.
+
+    Return:
+        A list of dictionary of Swan Instance CP configuration Info.
+    """
+    # Create Orchestrator instance.
+    so = Orchestrator(api_key=None, login=False)
+    # Retrieve avaliable resources. (CPs that are currently avaliable)
+    avaliable_resources = so.get_instance_resources(available=avaliability)
```

Comment:
Traceback (most recent call last):
  File "/Users/renhaozhang/Documents/github/work/SwanCloudInc/swan-tool-box/swan-sdk-test/quick-test.py", line 107, in <module>
    avaliable_resources = swan_orchestrator.get_instance_resources(available=avaliability)
                                                                             ^^^^^^^^^^^^
NameError: name 'avaliability' is not defined